ocdaction/profiles/models.py

```diff
@@ -2,9 +2,14 @@
 
 from django.db import models
 from custom_user.models import AbstractEmailUser
+import datetime
 
 class OCDActionUser(AbstractEmailUser):
 
     # basic info
     username = models.CharField(max_length=24, blank=True)
-    
\ No newline at end of file
+    date_birth = models.DateField('date of birth')
```

Comment:
Please provide a default value of null=True and set blank=true too, as when yo make migration you will otherwise have to provide a value each time.